helix-tui/src/buffer.rs

```diff
@@ -293,6 +312,9 @@ impl Buffer {
             index += width;
             x_offset += width;
         }
+        if ellipsis && x_offset - (x as usize) < string.as_ref().chars().count() {
```

Comment:
I don't think it is a good idea to count characters. Number of characters is not always the same as number of columns used. Might have to use `UnicodeStrWidth::width` for this.

Reply:
I used unicode_width and it seems to work correctly - could you check ? #375